WordPress/src/main/java/org/wordpress/android/ui/jetpack/scan/ScanListItemState.kt

```diff
@@ -0,0 +1,42 @@
+package org.wordpress.android.ui.jetpack.scan
+
+import androidx.annotation.DrawableRes
+import org.wordpress.android.R
+import org.wordpress.android.ui.jetpack.scan.ScanListItemState.ViewType.SCAN_STATE
+import org.wordpress.android.ui.utils.UiString
+import org.wordpress.android.ui.utils.UiString.UiStringRes
+
+sealed class ScanListItemState(val type: ViewType) {
+    open fun longId(): Long = hashCode().toLong()
```

Comment:
Question (❓ ): Is the `open` modifier required here?
Minor (🔍 ): Also, just as an FYI, the explicit `Long` type specification can be removed here.

Reply:
> Is the open modifier required here?

Not actually 

> the explicit Long type specification can be removed here.

Removed both in a576c37